lib/packageModules.js

```diff
@@ -139,7 +139,12 @@ module.exports = {
 
     // Copy artifacts to package location
     if (isIndividialPackaging.call(this)) {
-      _.forEach(functionNames, funcName => copyArtifactByName.call(this, funcName));
+      const nodeFunctionNames = _.filter(functionNames, funcName => {
```

Comment:
The same `nodeFunctionNames` would also need to be used below in line 165 (previously 160) to avoid serverless-webpack still overriding the artifact path of the given function.